projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.scss

```diff
@@ -1,6 +1,6 @@
 @import "../../styles/variables";
 
-:host {
+pa-filtered-checkbox-group {
```

Comment:
is this a class or a tag? (wondering if you forgot the `.` or if this component contains instance of itself)

Reply:
that's the tag

Reply:
The css is non encapsulated so it is global.
By enclosing it in `pa-filtered-checkbox-group {}` we make sure it only applies to the component.